org.manaplus.ManaPlus.yaml

```diff
@@ -0,0 +1,61 @@
+app-id: org.manaplus.ManaPlus
+runtime: org.freedesktop.Platform
+runtime-version: '18.08'
+sdk: org.freedesktop.Sdk
+command: manaplus
+rename-desktop-file: manaplus.desktop
+rename-appdata-file: manaplus.appdata.xml
+rename-icon: manaplus
```

Comment:
Always prefer to have a copy-icon: true just in case.

Reply:
The game seems to work fine without the old icon file being kept. :-)

Reply:
But thanks for the tip!

Reply:
Perfect